src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,31 @@
+import styled, { injectGlobal } from 'styled-components';
+
+injectGlobal`
+    @font-face {
+    font-family: 'Helveticons';
+    src: url('https://styleguide.dagbladet.no/fonts/helveticons.woff');
+  }
+`;
+
+const ArrowBullet = styled.a`
+    color: ${props => props.theme.colors.darkGrey};
```

Comment:
colors.type is usually used as a default text color

Reply:
Got you